richardaspinall/poker-react-node
# player sit action logic

### Description

Logic that allows end user to sit at a seat on a poker table.

### How to test

1. Start up the app
2. Click on one of the seats and check the console logs
3. Try clicking on another seat once you've already sat down
4. Try clicking on the same seat from another client once client A is already sitting down

### Task

[CU-86cttp4ur](https://app.clickup.com/t/86cttp4ur)
